solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRQParserPlugin.java

```diff
@@ -78,13 +78,18 @@ public Query parse() throws SyntaxError {
         throw new SolrException(SolrException.ErrorCode.BAD_REQUEST,
             "Must provide model in the request");
       }
-
+      
+      String flString1 = params.get(CommonParams.FL, CommonLTRParams.FEATURE_REQUEST); 
+      String flString2 = params.get(CommonParams.FL, CommonLTRParams.FV_REQUEST); 
+      boolean featuresRequested = (flString1.indexOf(CommonLTRParams.FEATURE_REQUEST) >= 0 || flString2.indexOf(CommonLTRParams.FV_REQUEST) >= 0) ? true : false;
```

Comment:
These are hard coded checks that will not work since the fv transformer can be named anything by the client in the config.  The transformer however, already adds a parameter to the requests context, see references of LOG_FEATURES_QUERY_PARAM.  Lower down there is a param extractFeatures and some processing on that which I think applies to what you're doing here